src/greedy_match.cpp

```diff
@@ -31,8 +28,8 @@ List greedy_string(SEXP radix, CharacterVector to_match){
       if(holding.size() == 0){
         holding.push_back(NA_STRING);
```

Comment:
I'll have to think about it. I was looking at it earlier and nothing immediately popped out to me, but possibly. My brain's stuck on other things atm.